<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authors: Aleksej Hyzha &amp;&amp; Dmytro Denkin | PI-191</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: rgb(170, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_vars">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Variables</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mono">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
qproperty-alignment: AlignCenter;
background-color:rgb(255, 255, 255);
border:4px solid;
border-radius:9px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}
QPushButton:hover {
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color: #2d89ef;
border-width: 2px;
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}</string>
    </property>
    <property name="text">
     <string>Monotonic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_samo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
qproperty-alignment: AlignCenter;
background-color:rgb(255, 255, 255);
border:4px solid;
border-radius:9px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}
QPushButton:hover {
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color: #2d89ef;
border-width: 2px;
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}</string>
    </property>
    <property name="text">
     <string>Self-Dual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save01">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
qproperty-alignment: AlignCenter;
background-color:rgb(255, 255, 255);
border:4px solid;
border-radius:9px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}
QPushButton:hover {
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color: #2d89ef;
border-width: 2px;
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}</string>
    </property>
    <property name="text">
     <string>0-preserving
1-preseving</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SOP">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>130</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
qproperty-alignment: AlignCenter;
background-color:rgb(255, 255, 255);
border:4px solid;
border-radius:9px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}
QPushButton:hover {
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color: #2d89ef;
border-width: 2px;
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}</string>
    </property>
    <property name="text">
     <string>SOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POS">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
qproperty-alignment: AlignCenter;
background-color:rgb(255, 255, 255);
border:4px solid;
border-radius:9px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}
QPushButton:hover {
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color: #2d89ef;
border-width: 2px;
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}</string>
    </property>
    <property name="text">
     <string>POS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Polinom">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>169</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
qproperty-alignment: AlignCenter;
background-color:rgb(255, 255, 255);
border:4px solid;
border-radius:9px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}
QPushButton:hover {
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color: #2d89ef;
border-width: 2px;
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}</string>
    </property>
    <property name="text">
     <string>Zhegalkin 
polynomial</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_ist">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget
{
border: 3px solid;
    border-color: rgb(193, 0, 0);
border-radius: 9px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_table">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Truth Table</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>392</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_2vars">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2_variables</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_3vars">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3_variables</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButton_4vars">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4_variables</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
